prime-router/docker-compose.postgres.yml

```diff
@@ -0,0 +1,16 @@
+# Build environment
```

Comment:
What is this?  How is this different from docker-compose.build.yml and why have the same config?

Reply:
this doesn't use a persistent volume, which I'm theorizing was at least a part of the problem.